rust/counter/README.md

```diff
@@ -0,0 +1,90 @@
+# Counter
+
+This is a Rust / Yew port of the Mokoto Counter example. 
+
+This example demonstrates a counter application. It uses an orthogonally
+persistent `counter` variable to store an arbitrary precision natural number
+that represents the current value of the counter.
+
+By using the Motoko keyword `stable` when declaring the `counter` variable,
+the value of this variable will automatically be preserved whenever your canister code is
+upgraded. Without the `stable` keyword, a variable is deemed `flexible`, and its value
+is reinitialized on every canister upgrade, i.e. whenever new code is deployed to the canister.
+
+To learn more about these features of Motoko, see:
+* https://sdk.dfinity.org/docs/language-guide/motoko.html#_orthogonal_persistence
+* https://sdk.dfinity.org/docs/language-guide/upgrades.html#_declaring_stable_variables
+
+## Introduction
+
+The application provides an interface that exposes the following methods:
+
+*  `set`, which sets the value of the counter;
+
+*  `inc`, which increments the value of the counter; and
+
+*  `get`, which gets the value of the counter.
+
+## Prerequisites
+
+Verify the following before running this demo:
+
+*  You have downloaded and installed the [DFINITY Canister
+   SDK](https://sdk.dfinity.org).
```

Comment:
```suggestion
   SDK](https://internetcomputer.org/docs/current/developer-docs/build/install-upgrade-remove).
```
SDK has moved

Reply:
DONE + updated link in Motoko counter example